.schema/config.schema.json

```diff
@@ -982,6 +982,50 @@
               "title": "Host",
               "description": "The network interface to listen on."
             },
+            "timeout": {
```

Comment:
If you take [this part](https://github.com/ory/oathkeeper/blob/e25501c1c39ec2f6fa93f72cd59ec571fd105f86/.schema/config.schema.json#L985) and make it a definition [(example)](https://github.com/ory/oathkeeper/blob/e25501c1c39ec2f6fa93f72cd59ec571fd105f86/.schema/config.schema.json#L7-L22) called `serverTimeout`, you can reference the definition both here and [there](https://github.com/ory/oathkeeper/blob/e25501c1c39ec2f6fa93f72cd59ec571fd105f86/.schema/config.schema.json#L985) using `#/definitions/serverTimeout`